### ISSUE TITLE
Web-based first_setup script

### ISSUE THREAD

**Issue by christianp** [numbas/editor#726]
It would be nice to have a web interface for the first_setup script. This would let you see all the questions in context, we could provide help text and links to documentation, and it'd be less forbidding for people who aren't used to the terminal.

I made a start on this, in a branch called `first_setup_web`. It contains a script `first_setup_web.py`, which includes all of the logic from the existing `first_setup.py`, plus a request handler for a web server. At the moment it displays all the existing questions, but it doesn't do anything when you submit.

The `do_POST` method needs to do some validation, then call `command.handle()`. If there are any errors it should show the form along with error messages. If it all works, it should show some instructions for what to do next - stop the setup script and then run the real server.

The command-line script fills in several of the questions with suitable values if you say yes to "is this for development?". The web interface should hide those questions, and fill in the appropriate values on the server side when the form is submitted.

It would be good if the script could first check that the required packages are installed before starting. You can do this by checking for `ImportError` when importing packages not in the standard library:

```
import sys
try:
    import django
except ImportError:
    print("The django package can't be found. Have you installed it, and have you activated the virtualenv?")
    sys.exit(1)
```

**Comment by christianp** [comment]
Done in 0e453e973e96fb24d4ce7bbc3885715e09ae0c41
